SocialCareCaseViewerApi/V1/Boundary/Requests/CreatePersonalRelationshipRequest.cs

```diff
@@ -16,6 +16,8 @@ public class CreatePersonalRelationshipRequest
         [JsonPropertyName("type")]
         public string Type { get; set; } = null!;
 
+        public long TypeId { get; set; }
```

Comment:
We might want to utilise `[JsonPropertyName("propertyName")] ` here, keeps it consistent with the rest of the class and makes it very easy to update the expected JSON body key without changing any variable names.

Reply:
Also just to confirm, are we always expecting TypeId to be provided? If not it should be made nullable.

Reply:
So `TypeId` won't be provided as part of the frontend request, instead it will be added by the use case for the gateway to use. Do you think that too cheeky? Basically wanted to avoid creating another class for the gateway method parameter... 👀 

But you're right we need to make it nullable, thanks!

Reply:
that's fine, just add the `[jsonIgnore]` attribute